overview.md

```diff
@@ -43,6 +43,20 @@ A Pulumi access token is required so that the Pulumi task can log you into your
 
 > The Pulumi Access Token is a sensitive value. Click on the padlock icon to mark it as a secret when you save it in your Pipeline variables or variable group.
 
+### Pulumi summary log as PR comments
+
+The Pulumi task supports adding PR comments containing the log output from the Pulumi command that was executed in your build pipeline.
+Your project's build service user will need additional permissions to perform that action. Follow these steps to grant the build service user the `Contribute to pull requests` permission:
```

Comment:
Oh, is it the case that users who don't enable PR comments won't require additional permissions? If that's the case, strike my above comment, we can release this as a minor version.

Reply:
Yeah that's right. However, there's a caveat. I've added a screenshot of the experience in the PR description. Basically, the extension needs to declare the required scopes in its manifest, as well as the user needs to grant explicit permissions. In other words, simply doing one of those two things is not enough.

I also think that a minor version should suffice. However, note that due to the change in the scope in the extension manifest file users will need to review and approve the permissions. Until they do that build agents will continue using the previously approved version of the task. So in essence, it won't auto-update until the scope change is approved. This is related to the screenshot I mentioned above.

Reply:
Oh, that makes a lot of sense. Thank you for the explanation. Sounds great, a minor version works well :)

Reply:
Once this PR is merged. I think we should run the "package" build with the [version override variable](https://github.com/pulumi/pulumi-az-pipelines-task/blob/master/package.azure-pipelines.yml#L38) set. That'll produce a package with the desired version rather than reading the current version from the marketplace and bumping the patch version. The pipelines in this repo are set to automatically bump patch versions as those are more common.